config.toml

```diff
@@ -138,13 +138,13 @@ time_format_default = "January 02, 2006 at 3:04 PM PST"
 description = "Production-Grade Container Orchestration"
 showedit = true
 
-latest = "v1.25"
+latest = "v1.26"
 
-fullversion = "v1.25.0"
-version = "v1.25"
-githubbranch = "main"
-docsbranch = "main"
-deprecated = false
+fullversion = "v1.26.0"
+version = "v1.26"
+githubbranch = "v1.26.0"
```

Comment:
I don't think the githubbranch, version, docsbranch, and deprecated need to be changed here. Refer to [this PR](https://github.com/kubernetes/website/pull/33412) for 1.24.